assets/chat/js/menus/ChatEmoteTooltip.js

```diff
@@ -8,12 +8,21 @@ export default class ChatEmoteTooltip extends ChatMenuFloating {
     this.emote = this.ui.find('.emote-image');
     this.name = this.ui.find('.emote-info .name');
     this.creator = this.ui.find('.emote-info .creator');
+    this.tier = this.ui.find('.emote-info .tier');
 
     this.chat.output.on('contextmenu', '.msg-chat .text .emote', (e) => {
       const emote = $(e.currentTarget).closest('.emote')[0].innerText;
+      const { creator, minimumSubTier } =
+        this.chat.emoteService.getEmote(emote) ?? {};
 
       this.name.text(emote);
-      this.setCreator(emote);
+
+      this.creator.text(creator);
+      this.creator[creator ? 'show' : 'hide']();
+
+      this.tier.text(`Tier ${minimumSubTier}`);
+      this.tier[minimumSubTier > 0 ? 'show' : 'hide']();
```

Comment:
Kind of a minor thing, but would you mind using setters for these two? I think setters are a great use case for when the visibility of an element depends only on its text value.